python/CMakeLists.txt

```diff
@@ -48,8 +48,10 @@ endif()
 
 rapids_cython_init()
 
+add_subdirectory(ucxx/examples)
 add_subdirectory(ucxx/_lib)
 
 if(DEFINED cython_lib_dir)
   rapids_cython_add_rpath_entries(TARGET ucxx PATHS "${cython_lib_dir}")
+  rapids_cython_add_rpath_entries(TARGET ucxx_examples PATHS "${cython_lib_dir}")
```

Comment:
Shouldn't need this.
```suggestion
```

Reply:
This is needed for a proper build/install of https://github.com/rapidsai/ucxx/pull/46/files/7e0ad1b7de36915eb3edf78898e1208e5eebcca6#diff-0cdf2e3fa52349932e0e3d2f84da5e2c9d3c664140dbe62c74db204c97ef4807.

Reply:
I think you're actually right here, tested locally without it and seemed to build correctly. I'll try removing it and see if CI passes too.